src/Symfony/Component/VarDumper/Tests/Command/ServerDumpCommandTest.php

```diff
@@ -28,6 +32,24 @@ public function testComplete(array $input, array $expectedSuggestions)
         $this->assertSame($expectedSuggestions, $tester->complete($input));
     }
 
+    public function testServerDumpSuccess()
+    {
+        $command = $this->createCommand();
+        $commandTester = new CommandTester($command);
+
+        $data = new Data([['my dump']]);
+        $input = base64_encode(serialize([$data, ['timestamp' => time(), 'source' => ['name' => 'sourceName', 'line' => 222, 'file' => 'myFile']]]))."\n";
+
+        $commandTester->setInputs([$input]);
+
+        $commandTester->execute(['--format' => 'html']);
```

Comment:
For this test, I can only use html because value of the dump doesn't be available in the command result. I seems that it be prompt in php stream not in the command.  
@ogizanagi Could you help me to add cli format in this test ?